opendcs-web-client-jetty/build.gradle

```diff
@@ -35,15 +41,51 @@ task startJetty(type: JavaExec) { ->
     mainClass = "org.opendcs.odcswebclient.jetty.Start"
 }
 
-task prepareStartScript(type: Copy) {
-    from './bin/start.sh'
-    into "${buildDir}/odcsapi-jetty-standalone/bin/"
-    filter(ReplaceTokens, tokens: ['jarName': jar.archiveFileName.get()])
+task copyWar(type: Copy) {
+    dependsOn ':opendcs-web-client:war' // depends on war task in this project
+    from project(':opendcs-web-client').tasks['war'] // get the war task from this project
+    into "${project.layout.getBuildDirectory().get().getAsFile().toString()}/bin"
+}
+
+task createStartScript() {
+    doLast {
+        //Make bin directory if it does not exist already.  buildDir is now deprecated.
+        File curBuildDir = project.layout.getBuildDirectory().get().getAsFile();
+        File binDir = new File(curBuildDir.toString(), "bin");
+        project.mkdir(binDir);
+        //Make the start.sh file.
+        new File(binDir, "start.sh").text = """
+#!/bin/bash
+cd `dirname \$0`/..
+java \$JAVA_OPTS -jar libs/${project.name}-${project.version}.jar \$JAVA_ARGS
+            """
+
+        //Make a config directory for config files to be built.
+        File configDir = new File(curBuildDir.toString(), "config");
+        project.mkdir(configDir);
+        //Make sample api config file.
+        new File(configDir, "sample_api.conf").text = """
+url=http://127.0.0.1
+port=8080
+context=odcsapi
+"""
+    }
+}
+
+task filepermission() {
+    doLast {
+        String start_path = "${buildDir}\\bin\\start.sh"
+        System.out.println("Here: " + start_path)
```

Comment:
Remove debug statement (or use the gradle logger)

Reply:
This function does not need to be in there.  Removing it.